docs/upgrades.md

```diff
@@ -62,6 +62,29 @@ If you want to manually control the upgrade procedure, you can use the variables
 `upgrade_node_confirm: true` - waiting to confirmation to upgrade next node
 `upgrade_node_pause_seconds: 60` - pause 60 seconds before upgrade next node
 
+## Node-based upgrade
+
+If you don't want to upgrade all nodes in one run, you can use `--limit` [patterns](https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html#patterns-and-ansible-playbook-flags).
```

Comment:
Could we describe what can be a main purpose of this upgrading way here?

Reply:
Example: we have in cluster 5 ingress nodes, and 100 nodes with application (bare metal cluster). In this case we want to upgrade nodes with application using `serial=10`. But for ingress nodes we want to disable them from dns first and then upgrade with `serial=1` one by one.

Reply:
I see, thanks for your explanation.